optimum/exporters/tasks.py

```diff
@@ -525,6 +525,10 @@ class TasksManager:
             "text-generation-with-past",
             onnx="CodeGenOnnxConfig",
         ),
+        "colpali": supported_tasks_mapping(
+            "feature-extraction",
+            onnx="ColPaliOnnxConfig",
+        ),
```

Comment:
> optimum-cli is currently not supported. To support it, ColPaliForRetrieval should be mapped in AutoModel in transformers, not in AutoModelForPreTraining as of now.

You should be able to bypass that using `_CUSTOM_CLASSES` by registering it as a custom class (same is done for pix2struct)